integration/generator/resources/linux_test_matrix.json

```diff
@@ -2,7 +2,7 @@
   {
     "os": "ubuntu",
     "username": "ubuntu",
-    "installAgentCommand": "sudo dpkg -i -E ./amazon-cloudwatch-agent.deb",
+    "installAgentCommand": "sudo apt-get install build-essential -y && sudo dpkg -i -E ./amazon-cloudwatch-agent.deb",
```

Comment:
Should the install build-essential be done at the ami builder pipeline level?
I think we should favor apt instead of apt-get.

Reply:
It should be at the AMI builder level. Where is that?

Reply:
Remind me to go over this Monday with you.

Reply:
```yaml
name: gcc-ubuntu
description: gcc-ubuntu
schemaVersion: 1.0

phases:
  - name: build
    steps:
      - name: gcc-ubuntu
        action: ExecuteBash
        inputs:
          commands:
            - sudo apt install build-essential -y

```

Created this new component and added it to the Ubuntu image recipe, and kicking off a new build. Tomorrow we can sync up and I can remove this command in the install.